src/pages/docs/reference/relay-point-address.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: RelayPointAddress object
+description: An object representing the location of a relay point.
+---
+
+Relay Point Address Object
+===============================================
+An object representing the address of a relay point.
+
+<Reference>
+  
+  <Field name="relay_point_id" type="string" required={true}>
+    <Description>
+      This should be the unique id the carrier uses for this relay point.
+    </Description>
+  </Field>
+
+  <Field name="company_name" type="string" required={false}>
+    <Description>
+        Company name, if applicable. Cannot include newline chars.
+    </Description>
+  </Field>
+
+  <Field name="address_lines[]" type="string[]" required={true}>
+    <Description>
+      Address fields separated by line. Up to 100 chars per line. Cannot include newline chars.
```

Comment:
"fields separated by line [...] Cannot include newline chars." seems a little unclear. Should this be something like "individual fields cannot include newline characters."?